src/components/organisms/AppRouter/AppRouter.jsx

```diff
@@ -81,7 +81,15 @@ const AppRouter = ({ defaultVenue }) => {
         />
         <Route
           path="/"
-          component={() => <Redirect to={venueLandingUrl(defaultVenue)} />}
+          component={() => (
+            <Redirect
+              to={
+                defaultVenue === "playa"
```

Comment:
let’s supply a `defaultRedirect` to AppRouter and move this logic there, so there’s only one place in the code with this special logic. So index.tsx would be similar to the existing code but bring the venueLandingUrl/venueInsideUrl into it, something like..

const defaultRedirect = projectId === “co-reality-map” ? venueLandingUrl(“kansassmittys”) : venueInsideUrl(“playa”)